docs/contributing/tests.md

```diff
@@ -1,42 +1,46 @@
 # Image Tests
 
-We greatly appreciate pull requests that extend the automated tests that vet the basic functionality
-of the Docker images.
+We greatly appreciate pull requests that extend the automated tests that vet the basic functionality of the Docker images.
 
 ## How the Tests Work
 
 GitHub Action executes `make build-test-all` against pull requests submitted to the `jupyter/docker-stacks` repository.
-This `make` command builds every docker image.
-After building each image, the `make` command executes `pytest` to run both image-specific tests like those in
-[base-notebook/test/](https://github.com/jupyter/docker-stacks/tree/master/base-notebook/test) and
-common tests defined in [test/](https://github.com/jupyter/docker-stacks/tree/master/test).
-Both kinds of tests make use of global [pytest fixtures](https://docs.pytest.org/en/latest/reference/fixtures.html)
-defined in the [conftest.py](https://github.com/jupyter/docker-stacks/blob/master/conftest.py) file at the root of the projects.
+This `make` command builds and then tests every docker image.
+
+We use `pytest` module to run tests on the image.
+`conftest.py` and `pytest.ini` in the `tests` folder define the environment in which tests are run.
+More info on `pytest` can be found [here](https://docs.pytest.org/en/latest/contents.html).
+
+All the actual test files are located in folders like `tests/<somestack>-notebook/`.
+
+```{note}
+If your test is located in `tests/<somestack>-notebook/`, it will be run against `jupyter/<somestack>-notebook` image and against all the images inherited from this image.
```

Comment:
```suggestion
If your test is located in `tests/<somestack>-notebook/`, it will be run against `jupyter/<somestack>-notebook` image and against all the [images inherited from this image](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html#image-relationships. 
```